src/agent/onefuzz-agent/src/debug/generic_regression.rs

```diff
@@ -0,0 +1,118 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+use crate::tasks::{
+    config::CommonConfig,
+    report::{libfuzzer_report, generic},
+    utils::parse_key_value,
+};
+use anyhow::Result;
+use clap::{App, Arg, SubCommand};
+use onefuzz::{blob::BlobContainerUrl, syncdir::SyncedDir};
+use std::{
+    collections::HashMap,
+    path::{Path, PathBuf},
+    sync::Arc,
+};
+use tokio::runtime::Runtime;
+use url::Url;
+use uuid::Uuid;
+
+async fn run_impl(input: String, config: Config) -> Result<()> {
```

Comment:
Where does `Config` come from?  I don't see it on an import

Reply:
Oh!  These are not included in `src/debug/mod.rs` or in `src/debug/cmd`.  Adding them exposed some compile errors

Reply:
removed the file. will add the debug/local capability in a separate PR